examples/clients/cli/README.md

```diff
@@ -20,7 +20,13 @@ you should see `> CONNECTED` in the terminal of running WASM, then you can conne
        {
            "server": "127.0.0.1",
            "server_port": 8388,
-           "password": "Test!23",
+           
+           // match the password in the config passed into WATER-cli [remove the comment when copying]
```

Comment:
Not a review opinion: Any idea why is this highlighted in a dark red background?

Reply:
I guess it's because json doesn't support comments? That's why I also added the `[remove the comment when copying]`

Reply:
Fair enough. 😅